mypy/modulefinder.py

```diff
@@ -443,10 +444,23 @@ def find_modules_recursive(self, module: str) -> List[BuildSource]:
         names = sorted(self.fscache.listdir(package_path))
         for name in names:
             # Skip certain names altogether
-            if name == '__pycache__' or name.startswith('.') or name.endswith('~'):
+            if (
+                name in ("__pycache__", "site-packages", "node_modules")
+                or name.startswith(".")
+                or name.endswith("~")
```

Comment:
I don't remember seeing directories with a `~` suffix, and files will be ignored based on the extension, so I think that this can be safely removed.